src/util/slateHelpers.jsx

```diff
@@ -196,12 +194,11 @@ export const paragraphRule = {
   // div handling with text in box (bodybox)
   deserialize(el, next) {
     if (el.tagName.toLowerCase() !== 'p') return;
-    const parent = el.parentElement ? el.parentElement.tagName.toLowerCase() : '';
-    const type = parent === 'li' ? 'list-text' : 'paragraph';
```

Comment:
Trur nesten vi må ha dette slik for at all tekst i et li-element skal håndteres av liste-pluginen. Men kanskje må det gjøres meir generellt. Altså at sjekk på om parent er `li` må inn fleire plasser for å unngå at ting mistes? Kanskje en generell regel?